app/models/case_log.rb

```diff
@@ -66,10 +67,15 @@ def form
   end
 
   def collection_start_year
```

Comment:
What about using this method as a getter for @start_year?
The risk I see is that someone could query `start_year` between the moment it was reset and the next call to `collection_start_year`